pages/news/2020-05-16-intro-outro-sections.html

```diff
@@ -0,0 +1,141 @@
+title: Intro & Outro Cues, Silence Detection, and AutoDJ Improvements
+author: Be.
+date: 2020-05-16 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+Do you use hotcues to mark the points you use to mix in and out of tracks? Have you wished AutoDJ knew to mix in and out at those points?
+In Mixxx 2.3, we have introduced new intro & outro cues for these purposes.
+These are not ordinary cue points. Unlike hotcues, and unlike intro and outro cues in other DJ software, they do not mark only one point. The intro & outro are sections and each is defined by two points.
+
+When analyzing a track, Mixxx detects where the first and last sounds are to make it easy to skip playing silent parts at the beginning and end of the track.
+The analyzer places the intro start point at the first sound and the outro end point at the last sound. The first and last sound are determined by the first time the signal rises above -60 dBFS and the last time it goes below -60 dBFS.
+
+<a href="/static/images/news/intro-outro/intro-start-outro-end.png">
+<img src="/static/images/news/intro-outro/intro-start-outro-end.png" style="max-height: 137px; max-width: 100%;" alt="A track with the automatically placed intro start and outro end cues"/>
+</a>
+
+It is up to you to decide where to place the intro end and outro start points.
+You can easily jump to the intro end and outro start points by pressing the buttons like hotcues, but that is not their primary purpose.
```

Comment:
I think this line should be separated from the previous one, they aren't really coherent.

Reply:
Also, you can jump to any of these points by pressing the respective buttons, not only "intro end and outro start"

Reply:
I simply removed that sentence.